File: documentation/user/en/query/ordering/examples/constant/attribute-set-in-filter.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					attributeInSet(
						"code", 
						"msi-gs66-10sf-stealth-1", 
						"apple-iphone-14-plus", 
						"lenovo-thinkpad-p14s-5"
					)
				),
				orderBy(
					attributeSetInFilter("code")
				),
				require(
					entityFetch(
						attributeContent("code")
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/ordering/examples/constant/attribute-set-in-filter.cs
EvitaResponse<SealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		AttributeInSet(
        			"code", 
        			"msi-gs66-10sf-stealth-1", 
        			"apple-iphone-14-plus", 
        			"lenovo-thinkpad-p14s-5"
        		)
        	),
        	OrderBy(
        		AttributeSetInFilter("code")
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code")
        		)
        	)
        )
	)
);